torch/_functorch/aot_autograd.py

```diff
@@ -138,86 +138,365 @@ def posthook(grad_input, grad_output):
         )
         node.register_hook(get_posthook(special_stack))
 
-# This class tells us about a user's forward output that is an alias.
-# It can be an alias of either a user forward input, of of a graph intermediate.
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+#
+# AOT Autograd contains a pretty non-trivial amount of logic to handle edge cases around aliasing and mutation
+# that are external to the graph (they show up as side effects in some way when you run the graph).
+#
+# Take a look at `test_aotdispatch.py TestAOTAutograd` for some examples functions and what they're compiled graphs looks like.
```

Comment:
uhh, you probably want to give a more specific file pointer, there are a lot of tests in that class